samples/pmic/npmx/buck/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _nPM1300 BUCK:
+
+nPM1300 BUCK
+############
+
+Overview
+********
+
+This sample demonstrates the usage of nPM1300 PMIC BUCKs.
+For testing connect voltage multimeter to VOUT1 (output voltage pin of BUCK1). In this example
+voltage can be changed only for BUCK1. BUCK2 voltage is set by default to 3.0V.
+In main.c file select your test case to be executed.
+You can also modify expected output voltages and selected GPIOs to control the system.
+
+Testcases
+---------
+
+Test set output BUCK voltage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set output voltage (simple)`.
+Voltage is set to 2.4 V (`NPMX_BUCK_VOLTAGE_2V4`) on buck 1.
+
+Test output voltage range
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set output voltage (advanced)`.
+Now only test_output_voltage() function is called.
+BUCK1 is starting from voltage 1.0 V (`NPMX_BUCK_VOLTAGE_1V0`).
+The output voltage will increase from the selected to the maximum voltage (3.3 V).
+
+Test retention voltage using GPIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set retention voltage of buck 1 with selected external pin`.
+Now only test_retention_voltage() function is called.
+GPIO1 (`NPMX_GPIO_INSTANCE_1`) is set to input mode and configured to be used as a retention input selector for buck 1 (`NPMX_BUCK_INSTANCE_1`).
+Normal mode voltage is set to 1.7 V (`NPMX_BUCK_VOLTAGE_1V7`) and retention voltage to 3.3 V (`NPMX_BUCK_VOLTAGE_3V3`).
+When you change the input state of the selected GPIO you should see voltage changes in the buck 1 output.
+
+Test enable BUCK using GPIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In `Testcase selection` select `Set enable mode of buck 1 with selected external pin`.
+Now only test_enable_bucks() function is called.
+GPIO3 (`NPMX_GPIO_INSTANCE_3`) is set input mode and configured to be used as enable mode selector for buck 1 (`NPMX_BUCK_INSTANCE_1`).
+The output voltage of buck 1 is set to 3.3 V (`NPMX_BUCK_VOLTAGE_3V3`).
+When you change the input state of the selected GPIO you should see voltage changes in the buck 1 output - from 0 V to previously selected voltage.
+
+Requirements
+************
+
+To run this sample, do the following:
+
+* Connect the nRF5340 or nRF52840 DK to a host device.
+* Open a COM port.
+* Connect nPM1300 EK with the chosen DK.
+
+Connections
+-----------
```

Comment:
```suggestion
Wiring
******
```

This entire section should go under the **Overview** one (as indicated in the template).